plugin/advisor/mysql/mysql_syntax_parser_test.go

```diff
@@ -0,0 +1,16 @@
+package mysql
+
+import "testing"
+
+// Test if parser support MySQL8 window function syntax.
```

Comment:
Seems to be better to put the comment in parser file, just above p.EnableWindowFunc(true)